test/avocado/testlib_avocado/seleniumlib.py

```diff
@@ -249,6 +250,34 @@ def check_box(self, element, checked=True):
             self.take_screenshot(fatal=False)
             raise SeleniumElementFailure('Unable to CHECKBOX element ({})'.format(e))
 
+    def select(self, element, by_text=None, by_value=None):
+        failure = "Select: too many tries"
+        if by_text and by_value:
+            raise AttributeError("unable to use by_text and by_value in selector")
+        elif (not by_text) and (not by_value):
+            raise AttributeError("You have to use some selector for select")
+        usedfunction = self.element_wait_functions[element] if element in self.element_wait_functions else None
+        for foo in range(self.default_try):
```

Comment:
I find 'foo' entertaining but I guess we could pick up a better name here.
(Regardless that it exists in other places as well)